uint16 actuators_data0
uint16 actuators_data1
uint16 actuators_data2
string R1D
string R1G
string V1
string D1
string R2D
string R2G
string V2
string D2
string ST1
string ST2
string ST3
string ST5
string ST4
string PI1
string PI2
string R11D
string R11G
string V11
string D11
string R12D
string R12G
string V12
string D12
string ST20
string ST21
string ST24
string ST23
string PI3
string PI4